services/mobile-frontend/src/components/shared/typography.js

```diff
@@ -0,0 +1,62 @@
+import { Colors } from 'react-native-paper';
+
+export const typographyStyles = {
```

Comment:
react-native-paper doesn't have any sort of typography stuff going on, so I made a simple one